fetch/range/resources/long-wav.py

```diff
@@ -50,11 +50,28 @@ def main(request, response):
     response.headers.set("Cache-Control", "no-cache")
 
     range_header = request.headers.get('Range', '')
+    range_header_match = range_header and re.search(r'^bytes=(\d*)-(\d*)$', range_header)
```

Comment:
So this is either None (or False?) or a regular expression result?

Reply:
Yep